### DISCUSSION TITLE
mozc-utils-gui not installed by default with ibus-mozc

### DISCUSSION THREAD

**Discussion by bertsons** [google/mozc#883] [Ideas]
I'm using Ubuntu 23.10 with ibus-mozc (installed via `sudo apt install ibus-mozc`).

When I go Settings -> Keyboard -> Input Sources -> Japanese (Mozc) -> Preferences, nothing appears.

To fix I have to install `sudo apt install mozc-utils-gui`

mozc-utils-gui should be installed by default with ibus-mozc as a dependency.

**Comment by yukawa** [comment]
Hi,

> mozc-utils-gui should be installed by default with ibus-mozc as a dependency.

To make it happen, you first need to convince the package maintainers in Debian project to move `mozc-utils-gui` from `Recommends` section to `Depends` section at Debian issue tracker (https://www.debian.org/Bugs/), then wait for the updated debian packages to be imported into new Ubuntu releases.

https://salsa.debian.org/debian/mozc/-/blob/8d9a97b018db14942121873f0585edc9806ef363/debian/control#L60-66
```
Depends:
 ${misc:Depends},
 ${shlibs:Depends},
 mozc-data,
 mozc-server (= ${binary:Version}),
 ibus (>= 1.2)
Recommends: mozc-utils-gui (= ${binary:Version})
```

Otherwise, you can specify `--install-recommends` so that `mozc-utils-gui` can also be installed automatically.
```
sudo apt install --install-recommends ibus-mozc
```

Mozc project itself does not own/maintain these Debian/Ubuntu packages, and we cannot make it happen by updating code under https://github.com/google/mozc. What needs to be updated is https://salsa.debian.org/debian/mozc/.
